src/forms.rs

```diff
@@ -76,8 +77,211 @@ impl<'a> Field<'a> {
     }
 }
 
+/// Only permissible on text fields.
+impl<'a> Field<'a> {
+    // TODO: the spec likely means the equivalent of unicode graphemes here
+    //       for characters
+
+    /// Write the `/MaxLen` attribute to set the maximum length of the field's
+    /// text in characters. Only permissible on text fields.
+    pub fn text_max_len(&mut self, len: i32) -> &mut Self {
+        self.dict.pair(Name(b"MaxLen"), len);
+        self
+    }
+
+    /// Write the `/V` attribute to set the value of this text field.
+    /// Only permissible on text fields.
+    pub fn text_value(&mut self, value: TextStr) -> &mut Self {
+        self.dict.pair(Name(b"V"), value);
+        self
+    }
+
+    /// Start writing the `/DV` attribute to set the default value of this text
+    /// field. Only permissible on text fields.
+    pub fn text_default_value(&mut self, value: TextStr) -> &mut Self {
+        self.dict.pair(Name(b"DV"), value);
+        self
+    }
+}
+
+/// Only permissible on fields containing variable text.
+impl<'a> Field<'a> {
+    /// Write the `/DA` attribute containing a sequence of valid page-content
+    /// graphics or text state operators that define such properties as the
+    /// field's text size and colour. Only permissible on fields containing
+    /// variable text.
+    pub fn vartext_default_appearance(&mut self, appearance: Str) -> &mut Self {
+        self.dict.pair(Name(b"DA"), appearance);
+        self
+    }
```

Comment:
Would it be more convenient if this returned a writer for a content stream?

Reply:
I honestly don't know, the spec simply said it's a string, so I figured `Str` would be it, is a writer for this kind of text operators available already, or should I add one?

Reply:
Okay nvm its probably too complicated